README.md

```diff
@@ -11,8 +11,24 @@
 python3 -m install pytraccar
 ```
 
+
+## Usage
+
 Look at the file `example.py` for a usage example.
 
+`ApiClient` can be initialized as follows:
+
+Required args:
+* `host`: string hostname
+* `token`: string, see [the API doc][api-doc] for details
+* `client_session`: `aiohttp.ClientSession`; see example.py
+
+Optional kwargs:
+* `port`: integer, default `8082`
+* `ssl`: boolean, default `False`
+* `verify_ssl`: boolean, default `True`
+* `ws_heartbeat`: integer, default `120`
```

Comment:
This is not the place for it. If more details are needed, that needs to be added to the docstring of `ApiClient`, not here. If added here, it will be forgotten to update when it changes, and it will get out of sync.

The docstring is where a developer first looks, so that is where it needs to be if needed.

Reply:
Replaced by #351